src/api/app/views/webui/image_templates/index.html.haml

```diff
@@ -40,7 +40,7 @@
           (Maximum of 200 characters, no blank, /, :, - or ; characters)
         %br
         = text_field_tag(:target_package, first_template, class: 'name_appliance_form', required: true)
-        = submit_tag('Create appliance', class: 'create_appliance')
+        = submit_tag('Create appliance', class: 'create_appliance', disabled: User.current.is_nobody? ? true: nil)
```

Comment:
`User.current.is_nobody?` already returns true/false so i dont think `? true: nil` is necessary here?